cephfs/cephfs.go

```diff
@@ -155,8 +167,25 @@ func (mount *MountInfo) Unmount() error {
 // Implements:
 //  int ceph_release(struct ceph_mount_info *cmount);
 func (mount *MountInfo) Release() error {
+	if !mount.initialized {
+		return nil
+	}
 	ret := C.ceph_release(mount.mount)
-	return getError(ret)
+	if err := getError(ret); err != nil {
+		return err
+	}
+	mount.initialized = false
+	return nil
+}
+
+// SetMountPerms applies the given UserPerm to the mount object, which it will
+// then use to define the connection's ownership credentials.
+// This function must be called after Init but before Mount.
+//
+// Implements:
+//  int ceph_mount_perms_set(struct ceph_mount_info *cmount, UserPerm *perm);
+func (mount *MountInfo) SetMountPerms(perm *UserPerm) error {
+	return getError(C.ceph_mount_perms_set(mount.mount, perm.userPerm))
```

Comment:
[Travis CI job](https://travis-ci.org/github/ceph/go-ceph/jobs/666583663) failed with this:
```
cephfs/cephfs.go:188:18: could not determine kind of name for C.ceph_mount_perms_set
```
This seems to have been introduced with Mimic.

Reply:
Yeah. I was hoping to get the PR this depends on, #204, reviewed and merged first and then I planned to get back to this one and fix the issues. However, the CI issues distracted me.

Reply:
Fixed.